stdlib/3/tkinter/__init__.pyi

```diff
@@ -650,6 +650,7 @@ class Pack:
         padx: Union[_ScreenUnits, Tuple[_ScreenUnits, _ScreenUnits]] = ...,
         pady: Union[_ScreenUnits, Tuple[_ScreenUnits, _ScreenUnits]] = ...,
         in_: Misc = ...,
+        **kw: Any,
```

Comment:
Stubs typically don't contain much comments, but a comment explaining why there is `**kw` and mentioning the issue number would be useful.